things/clients/api/transport.go

```diff
@@ -278,23 +278,23 @@ func decodeChangeClientStatus(_ context.Context, r *http.Request) (interface{},
 func decodeListMembersRequest(_ context.Context, r *http.Request) (interface{}, error) {
 	s, err := apiutil.ReadStringQuery(r, api.StatusKey, api.DefClientStatus)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, apiutil.ErrValidation)
```

Comment:
check on your `errors.wrap`

Reply:
sorry, I don't understand this one 😅

Reply:
It's been answered on the previous comment